app/ui/browser-modern/index.js

```diff
@@ -43,3 +44,10 @@ const userAgentClient = setupUserAgentClient({
 
 setupFrontend({ store, userAgentClient });
 setupIPCCommunication({ store, userAgentClient });
+
+// Fired on the first browser window when default browser is not Tofino.
+ipcRenderer.once('should-set-default-browser', () => {
+  if (confirm('Would you like to be Tofino your default browser?')) {
```

Comment:
This should be an UI effect if we really want to write it properly.

On the `should-set-default-browser`, an CONFIRM_DEFAULT_BROWSER effect should be dispatched, handled in the UI sagas module, which on confirmation sends a `set-default-browser` ipc message.